inference-engine/tests/ie_test_utils/functional_test_utils/low_precision_transformations/layer_transformation.cpp

```diff
@@ -23,43 +23,70 @@
+    return InferenceEngine::details::LayerTransformation::Params(
+        false,
+        true,
+        true,
+        InferenceEngine::details::LayerTransformation::QuantizedTensorAlignment::None,
+        InferenceEngine::details::LayerTransformation::QuantizedTensorAlignment::None,
+        false,
+        true,
+        true,
+        { InferenceEngine::Precision::U8 },
+        { InferenceEngine::Precision::I8 });
+}
 
-    InferenceEngine::details::LowPrecisionTransformer LayerTransformation::getLowPrecisionTransformer(
-        const InferenceEngine::details::LayerTransformation::Params& params) const {
-        InferenceEngine::details::LowPrecisionTransformer transformer(getLowPrecisionTransformations(params));
-        return transformer;
-    }
+InferenceEngine::details::LayerTransformation::Params LayerTransformationParamsFactory::createParamU8U8() {
+    return InferenceEngine::details::LayerTransformation::Params(
+        false,
+        true,
+        true,
+        InferenceEngine::details::LayerTransformation::QuantizedTensorAlignment::None,
+        InferenceEngine::details::LayerTransformation::QuantizedTensorAlignment::None,
+        false,
+        true,
+        true,
+        { InferenceEngine::Precision::U8 },
+        { InferenceEngine::Precision::U8 });
+}
 
-    InferenceEngine::CNNNetwork LayerTransformation::transform() {
-        InferenceEngine::details::CNNNetworkImplPtr cnnNetworkImp = cloneNet(InferenceEngine::CNNNetwork(function));
+InferenceEngine::details::LayerTransformation::Params LayerTransformationParamsFactory::createParamI8I8() {
+    return InferenceEngine::details::LayerTransformation::Params(
+        false,
+        true,
+        true,
+        InferenceEngine::details::LayerTransformation::QuantizedTensorAlignment::None,
+        InferenceEngine::details::LayerTransformation::QuantizedTensorAlignment::None,
+        false,
+        true,
+        true,
+        { InferenceEngine::Precision::I8 },
+        { InferenceEngine::Precision::I8 });
+}
 
-        InferenceEngine::details::LayerTransformation::Params params = InferenceEngine::details::LayerTransformation::Params(
-            true,  // updatePrecisions
-            true,  // quantizeOutputs
-            true,  // weightsToConst
-            InferenceEngine::details::LayerTransformation::QuantizedTensorAlignment::UpdateLevel,  // quantizedTensorAlignmentOnActivations
-            InferenceEngine::details::LayerTransformation::QuantizedTensorAlignment::None,  // quantizedTensorAlignmentOnWeights
-            true,  // roundQuantizedValues
-            true,  // updateBiases
-            true); // supportAsymmetricQuantization
-        auto transformer = getLowPrecisionTransformer(params);
-        transformer.transform(*cnnNetworkImp);
+InferenceEngine::details::LayerTransformation::Params LayerTransformationParamsFactory::createParamCpu() {
```

Comment:
As I can see, It's code duplication. And more, I think it's bad idea to add parameters per HW

Reply:
good comment, thanks: 32278: [LPT] [TEST] Move hardware specific logic to parent classes